nx.json

```diff
@@ -74,9 +83,9 @@
     }
   },
   "plugins": [
-    "./examples/e2e/cli-e2e-original/tooling/original.plugin.ts",
```

Comment:
@BioPhoton Maybe you still want to keep the original plugin?

Reply:
The original plugin as well as the code-pushup reports for the original plugins should stay in the code.
This is important as we want to have comparison measures.